analysis/topEFT/topeft.py

```diff
@@ -275,7 +275,8 @@ def process(self, events):
 
         # Define the lists of systematics we include
         obj_correction_syst_lst = [
-            f'JER_{year}Up',f'JER_{year}Down','JESUp','JESDown' # Systs that affect the kinematics of objects
+            f'JER_{year}Up',f'JER_{year}Down', # Systs that affect the kinematics of objects
+            'JES_FlavorQCDUp', 'JES_RelativeBalUp', 'JES_HFUp', 'JES_BBEC1Up', 'JES_EC2Up', 'JES_AbsoluteUp', f'JES_BBEC1_{year}Up', f'JES_EC2_{year}Up', f'JES_Absolute_{year}Up', f'JES_HF_{year}Up', f'JES_RelativeSample_{year}Up','JES_FlavorQCDDown', 'JES_RelativeBalDown', 'JES_HFDown', 'JES_BBEC1Down', 'JES_EC2Down', 'JES_AbsoluteDown', f'JES_BBEC1_{year}Down', f'JES_EC2_{year}Down', f'JES_Absolute_{year}Down', f'JES_HF_{year}Down', f'JES_RelativeSample_{year}Down' # Systs that affect the kinematics of objects
```

Comment:
Here I think the uncorrelated ones should be  `f'JES_X__{year[0:4]}Up'` (as in my PR) instead of `f 'JES_X__{year}Up' to avoid the issue when year=='2016APV'

Reply:
The only reason I did it like this was so we would still have the `2016APV` events saved as `2016APV`. The corrections script properly loads the `2016` corrections.

Reply:
I see, but then how do you consider 'JES_X_2016' and 'JES_X_2016' as fully correlated?